.github/workflows/equinix_k8s_flow.yml

```diff
@@ -89,12 +89,14 @@ jobs:
       - name: Install Prometheus
         run: |
           git clone --depth 1 https://github.com/prometheus-operator/kube-prometheus; cd kube-prometheus;
-          kubectl apply --server-side -f manifests/setup
+          kubectl apply --server-side --validate=false -f manifests/setup
```

Comment:
looking at the failure this seems to be needed... locally i don't get a openapi error, not sure if you have seen this error before @rootfs `error validating "manifests/setup/0alertmanagerConfigCustomResourceDefinition.yaml": error validating data: failed to download openapi: Get "http://localhost:8080/openapi/v2?timeout=32s": dial tcp [::1]:8080: connect: connection refused; if you choose to ignore these errors, turn validation off with --validate=false`

Reply:
yes, that's because the kubeconfig is not installed on the runner in this step yet, it is installed in the `Test-k8s` job